packages/js/product-editor/src/utils/get-product-title.ts

```diff
@@ -3,7 +3,7 @@
  */
 import { __ } from '@wordpress/i18n';
 
-export const AUTO_DRAFT_NAME = 'AUTO-DRAFT';
+export const AUTO_DRAFT_NAME = 'Auto Draft';
```

Comment:
The other auto-draft records (from core) have the name `Auto Draft` instead of `AUTO_DRAFT`, so I updated this to match. Glad to revert if we prefer the capitals.

Reply:
I actually didn't see this in the current changes, I would prefer the `Auto Draft` it seems a little nicer.